src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -664,6 +664,15 @@ export default (accessToken, userID, opts) => {
       {},
       { headers: true }
     ),
+    partnerSearchCriteriaSingleLoader: gravityLoader<
```

Comment:
(non-blocking): maybe just `partnerSearchCriteriaLoader` (implies singular)

Reply:
that works, let me rename it's plural partner to `partnerSearchCriteriasLoader`